### PR TITLE
Disable CSS transitions when dark mode

### PR DESCRIPTION
**Pull request by AVGVSTVS96**
### TL;DR

This PR introduces a function that disables CSS transitions when toggling between light and dark themes.

### What changed?

A `disableTransitions` function has been added in `ModeToggle.tsx` and `ThemeToggle.astro` files to temporarily disable CSS transitions whenever the theme is switched, providing a smoother experience when switching between dark and light modes.

### How to test?

To test, you can switch between light and dark modes in the user interface and observe that transitions on elements don't appear during mode changes, but they do appear when they're intended to, i.e. when a button is hovered.

---

Update scripts, add `disableTransitions()`  to both mode toggle component implementations -`ModeToggle.astro` and `ThemeToggle.tsx`
 
This prevents the transition from being visible when toggling light and dark mode.

### CODE REVIEW

**Comment by AVGVSTVS96** [comment]
* **#269** <a href="https://app.graphite.dev/github/pr/AVGVSTVS96/astroSite/269?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @AVGVSTVS96 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>
